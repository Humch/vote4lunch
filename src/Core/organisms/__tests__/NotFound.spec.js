/* eslint-disable no-undef */
import React from 'react';

import renderer from 'react-test-renderer';

import { NotFound } from '../..';


describe('NotFound Snapshot', () => {
  it('capturing Snapshot of NotFound', () => {
    const renderedValue = renderer.create(
      <NotFound />
    ).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
