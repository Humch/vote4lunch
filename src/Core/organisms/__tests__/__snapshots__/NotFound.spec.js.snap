// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Snapshot capturing Snapshot of NotFound 1`] = `
<div
  className="NotFound row"
>
  <div
    className="col"
  >
    <h1
      className="text-center text-white mt-5 display-1"
    >
      404
    </h1>
  </div>
</div>
`;
